src/db/prepopulate.js

```diff
@@ -0,0 +1,306 @@
+'use strict';
+
+var Event = require('../models/event'),
+	User = require('../models/user');
+
+var userIds = [];
+
+function populateUsers() {
+	// Used http://www.fakemailgenerator.com to get emails
+	var user = new User({
+		userId: 'dc8afe3b4640421da3f1b0b4ce91a849',
+		firstName: 'Richard',
+		lastName: 'Gascon',
+		dateCreated: new Date('08/22/2016'),
+		email: 'richardegascon@armyspy.com',
+		profilePhoto: 'https://static.pexels.com/photos/1336/black-and-white-man-person-eyes.jpg',
+		refreshToken: 'xG1sw4vV0LymwB4cc9B8R8UO3FPOJ6vhJ59vK7e0UYpCFIFZAR'
+	});
+	user.save(function(err, data) {
+		console.log(`Saved ${data.firstName} ${data.lastName}`);
+		userIds.push(data._id);
+	});
+
+	var user2 = new User({
+		userId: '14471c78d9f64cfc892f5112118a82f0',
+		firstName: 'Barbra',
```

Comment:
I actually got it from a random name generator, and that is what the email had, so I left it. I noticed that though, lol.